rules/codelingo/effective-go/initialize-using-composite-literal/codelingo.yaml

```diff
@@ -1,5 +1,12 @@
-tenets:
-  - name:  initialize-using-composite-literal
+funcs:
+  - name: isOneMore
```

Comment:
can the function name be a lil more descriptive? something like: isDirectSibling or isNextSibling.